ledger-tool/src/main.rs

```diff
@@ -1381,6 +1443,27 @@ fn main() {
                 .about("Output statistics in JSON format about \
                         all column families in the ledger rocksdb")
         )
+        .subcommand(
+            SubCommand::with_name("compute-cost")
```

Comment:
It would be nice to sneak a "slot" in the subcommand name somewhere, perhaps:
```suggestion
            SubCommand::with_name("compute-slot-cost")
```